linux/nvme.h

```diff
@@ -418,8 +419,13 @@ struct nvme_id_ns {
 };
 
 struct nvme_iocs_vector {
-	__le64	nvm  : 1;
-	__le64	rsvd : 63;
+	union {
+		__le64 nvm   : 1;
+		__le64 rsvd1 : 1;
+		__le64 zoned : 1;
+		__le64 rsvd3 : 61;
```

Comment:
We can't use bitfields in a program where bit order matters because there's no guarantee from C what order it will lay them out. C99 reference, Section 6.7.2 paragraph 10:

"The order of allocation of bit-fields within a unit (high-order to low-order or low-order to high-order) is implementation-defined."